src/inc_gauged_raz.rs

```diff
@@ -37,6 +37,7 @@ const DEFAULT_SECTION_CAPACITY: usize = 500;
 /// The data stored in the tree structure of the RAZ.
 #[derive(PartialEq,Eq,Debug,Hash,Clone)]
 enum TreeData<E:Debug+Clone+Eq+Hash, M:RazMeta<E>> {
+	Dummy, // used when rebuild is certain
```

Comment:
OK.  Thanks for the clarification.  I still don't understanding the details of the rebuilding algorithm, but I follow what you said above.